### PR TITLE
Update tools.rst

### PR DESCRIPTION
**Pull request by mwalls**
Trying to fix more links. Hope the ones I've done are ok. But there are some broken ones here I don't know how to fix. (data, original metadata, angle and others).
I'll try and find out unless someone else can see immediately...

### CODE REVIEW

File: doc/user_guide/tools.rst [hunk 1]
```diff
@@ -58,12 +58,12 @@ examples, the data is stored in a numpy array in the
 :py:attr:`~.signal.BaseSignal.original_metadata` attribute, the mapped parameters
 in the :py:attr:`~.signal.BaseSignal.metadata` attribute and the axes
 information (including calibration) can be accessed (and modified) in the
-:py:attr:`~.signal.BaseSignal.axes_manager` attribute.
+:py:class:`~.axes.:py:class:`~.axes.AxesManager`` attribute.
```
Comment (francisco-dlp) [inline doc/user_guide/tools.rst]: ``py:class:`~.axes.AxesManager``  instead?

File: doc/user_guide/tools.rst [hunk 2]
```diff
@@ -58,12 +58,12 @@ examples, the data is stored in a numpy array in the
...
 
 Signal initialization
 ---------------------
 
-Many of the values in the :py:attr:`~.signal.BaseSignal.axes_manager` can be
+Many of the values in the :py:class:`~.axes.:py:class:`~.axes.AxesManager`` can be
```
Comment (francisco-dlp) [inline doc/user_guide/tools.rst]: As above

**Review by francisco-dlp — CHANGES_REQUESTED** [review]
Thanks! Just a minor issue to fix.

File: doc/user_guide/tools.rst (hunk 2 above)
Comment (mwalls) [reply doc/user_guide/tools.rst]: Actally, maybe I somehow changed that last line myself? It seems I did. I'll do it again Sorry.

**Comment by francisco-dlp** [comment]
Thanks!

**Pull request merged**